### PR TITLE
Move nci_s2_upload_s3 DAG to Sandbox/Processing cluster 

### PR DESCRIPTION
**Pull request by sachitrajbhandari**
Updated `nci_s2_upload_s3` DAG with prod credentials to run in Sandbox/Processing cluster

### CODE REVIEW

File: dags/nci_s2_upload_s3.py [hunk 1]
```diff
@@ -102,8 +61,8 @@ def get_parameter_value_from_variable(_dag_config, parameter_name, default_value
     'nci_s2_upload_s3',
     doc_md=__doc__,
     default_args=default_args,
-    catchup=get_parameter_value_from_variable(dag_config, 'catchup', False),
-    schedule_interval=get_parameter_value_from_variable(dag_config, 'schedule_interval', None),
+    catchup=True,
```
Comment (omad) [inline dags/nci_s2_upload_s3.py:64]: Do we actually want `catchup` turned on? I thought we'd be better off creating them from the CLI or by creating DAGRun's in the web UI.

File: dags/nci_s2_upload_s3.py [hunk 2]
```diff
@@ -115,7 +74,7 @@ def get_parameter_value_from_variable(_dag_config, parameter_name, default_value
     # '/g/data/v10/work/s2_nbar_rolling_archive/<current_datetime>_<end_date>_<num_days>'
     COMMON = """
             {% set work_dir = '/g/data/v10/work/s2_nbar_rolling_archive/' 
-            + execution_date.strftime('%FT%H%M') + '_'+ yesterday_ds +'_' + params.numdays  -%}
+            + execution_date.strftime('%FT%H%M') + '_'+ ds +'_' + params.numdays  -%}
             """
     create_nci_work_dir = SSHOperator(
```
Comment (omad) [inline dags/nci_s2_upload_s3.py:79]: This entire Operator can be skipped by setting `create_intermediate_dirs=True` on the SFTP Operator.

File: dags/nci_s2_upload_s3.py [hunk 3]
```diff
@@ -156,13 +115,13 @@ def get_parameter_value_from_variable(_dag_config, parameter_name, default_value
 
             python3 '{{ ti.xcom_pull(key='work_dir') }}/s2_to_s3_rolling.py' \
```
Comment (omad) [inline dags/nci_s2_upload_s3.py:116]: It's not necessary to use XCom here, either pass a parameter, or prepend the `COMMON` string to this template and use the variable directly.

File: dags/nci_s2_upload_s3.py (hunk 1 above)
Comment (sachitrajbhandari) [reply dags/nci_s2_upload_s3.py:64]: @omad  `catchup` is turned on to run daily DAG from `start_date` to `end_date`. Can we create a DAG Run to schedule @daily from `start_date` to `end_date` in the web UI?

**Pull request closed without merging**